src/main/java/org/tron/common/runtime/vm/program/Program.java

```diff
@@ -494,7 +494,7 @@ public void createContract(DataWord value, DataWord memStart, DataWord memSize)
 
     increaseNonce();
     // [5] COOK THE INVOKE AND EXECUTE
-    InternalTransaction internalTx = addInternalTx(null, senderAddress, null, endowment,
+    InternalTransaction internalTx = addInternalTx(null, senderAddress, newAddress, endowment,
         programCode, "create", nonce);
     long vmStartInUs = System.nanoTime() / 1000;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Assign this magic number 1000 to a well-named constant, and use the constant instead. ([squid:S109](https://sonarcloud.io/organizations/tron-zhaohong/rules#rule_key=squid%3AS109))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=java-tron&pullRequest=1633&issues=AWak_C1ndPPRVdwyhXJY&open=AWak_C1ndPPRVdwyhXJY)

[ISSUE_KEY]: AWak_C1ndPPRVdwyhXJY